core/chains/evm/logpoller/orm.go

```diff
@@ -189,7 +189,12 @@ func (o *ORM) InsertLogs(logs []Log, qopts ...pg.QOpt) error {
 (:evm_chain_id, :log_index, :block_hash, :block_number, :block_timestamp, :address, :event_sig, :topics, :tx_hash, :data, NOW()) ON CONFLICT DO NOTHING`, logs[start:end])
 
 		if err != nil {
```

Comment:
Note that we will overload the DB before `batchInsertSize` gets small enough. Using a relatively small timeout can help speed it up.